### ISSUE TITLE
How to import as a module

### ISSUE THREAD

**Issue by azazdeaz** [duckietown/apriltags3-py#6]
Hey, thanks a lot for this binding, works great! :tada:

I'm not sure how should I use it as a dependency. Currently, I clone the repo next to my sources and patch it with an `__init__.py` so I can import the `apriltags3.py`, but this doesn't feel like the best way (i don't have much experience with python).

The [pupil fork](https://github.com/pupil-labs/apriltags) has a nice scikit-build based solution to turn this into a pip installable package. It'd be great to have something similar here.

**Comment by AleksandarPetrov** [comment]
@azazdeaz indeed, adding a `setup.py` should solve the problem. Just out of curiosity, why are you not using the pupil fork?

**Comment by azazdeaz** [comment]
I'm using that for now, but they added this line to the readme: 
> ...specifically adjusted to work with the pupil-labs software...

So it doesn't feel like a good long term solution :)
